src/main/java/com/amazon/opendistro/elasticsearch/performanceanalyzer/metricsdb/MetricsDB.java

```diff
@@ -84,6 +89,21 @@ public MetricsDB(long windowStartTime) throws Exception {
     create = DSL.using(conn, SQLDialect.SQLITE);
   }
 
+  /**
+   * Returns a MetricsDB handle associated with an existing metricsdb file.
```

Comment:
Will calling it the latest metricDB file be more appropriate ?

Reply:
It would not. There may be 30 metricsdb files on disk. The user would call this method with the timestamp associated with one of them, and this method would return a MetricsDB handle to that file.